RetailCoder.VBE/UI/Command/ExportAllCommand.cs

```diff
@@ -0,0 +1,77 @@
+using System.IO;
+using System.Windows.Forms;
+using NLog;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using Rubberduck.Navigation.CodeExplorer;
+using Rubberduck.UI.CodeExplorer.Commands;
+
+namespace Rubberduck.UI.Command
+{
+    [CodeExplorerCommand]
+    public class ExportAllCommand : CommandBase 
```

Comment:
I can look into it and add a keyboard shortcut at a later date.  Personally, I'd like to get this added to `next` sooner than later so that I know it ships with the next release.